src/commands/doctor.ts

```diff
@@ -110,6 +110,11 @@ module.exports = {
       info('')
       info(colors.cyan('iOS'))
       table([[column1('xcode'), column2(xcodeVersion)]])
+
+      const cocoaPodsPath = which('pod')
```

Comment:
Can you just check `if (cocoaPodsPath) {` for the next block so you can print out that it isn't available instead of crashing?

Reply:
```typescript
const cocoaPodsPath = which('pod')
const cocoaPodsVersion = cocoaPodsPath ? await run('pod --version', { trim: true }) : "Not installed"
table([[column1('cocoapods'), column2(cocoaPodsVersion), column3(cocoaPodsPath)]])
```

Reply:
^ That would work, actually

Reply:
Should be a little less naive now